### PR TITLE
posix: Add /proc/uptime

### PR DESCRIPTION
**Pull request by Dennisbonke**

### CODE REVIEW

File: posix/subsystem/src/clock.cpp [hunk 1]
```diff
@@ -98,4 +98,30 @@ struct timespec getRealtime() {
...
+
+	// Finish the seqlock read.
+	__atomic_thread_fence(__ATOMIC_ACQUIRE);
+	assert(__atomic_load_n(&page->seqlock, __ATOMIC_RELAXED) == seqlock);
+
+	// Calculate the current time.
+	uint64_t now;
+	HEL_CHECK(helGetClock(&now));
+
+	int64_t timeSinceBoot = now - boot;
```
[the remark this answers is not in the document]
Comment (Dennisbonke) [reply posix/subsystem/src/clock.cpp:119]: I didn't know that. Will rework.

File: posix/subsystem/src/procfs.cpp [hunk 2]
```diff
@@ -334,6 +336,21 @@ async::result<frg::expected<Error>> DirectoryNode::unlink(std::string name) {
 	co_return frg::expected<Error>{};
 }
 
+async::result<std::string> UptimeNode::show() {
+	auto uptime = clk::getTimeSinceBoot();
+	// See man 5 proc for more details.
+	// Based on the man page from Linux man-pages 6.01, updated on 2022-10-09.
+	std::stringstream stream;
+	stream << uptime.tv_sec << "." << (uptime.tv_nsec / 10000000) << "\n";
```
Comment (qookei) [inline posix/subsystem/src/procfs.cpp:344]: This is formatted wrong? assume `uptime.tv_nsec / 10000000` is <10, then this prints `N.M` and not `N.0M` (so 10x the proper value after the decimal point)

Comment (no92) [reply posix/subsystem/src/procfs.cpp:344]: Yeah, generally procfs and sysfs `show()` functions should use snprintf etc for formatting output.

Comment (qookei) [reply posix/subsystem/src/procfs.cpp:344]: Or, even better, `frg::fmt` :^)

**Review by ElectrodeYT — APPROVED** [review]
LGTM

**Review by qookei — APPROVED** [review]

**Pull request merged**